tests/kernel/threads/dynamic_thread_stack/src/main.c

```diff
@@ -114,10 +114,27 @@ ZTEST(dynamic_thread_stack, test_dynamic_thread_stack_alloc)
 	/* spwan our threads */
 	for (size_t i = 0; i < N; ++i) {
 		tflag[i] = false;
+#ifdef CONFIG_XTENSA
```

Comment:
with https://github.com/zephyrproject-rtos/zephyr/pull/68140, this will not be needed :)

Reply:
was hoping for that :)